qa-dcp.planx-pla.net/portal/gitops.json

```diff
@@ -325,6 +325,7 @@
     },
     "guppyConfig": {
       "dataType": "subject",
+      "tierAccessLevel": "libre",
```

Comment:
this file doesn't need changes -- testing should be against site-wide tiered access